src/coordinator/tsdb/remote/codecs.go

```diff
@@ -40,6 +40,17 @@ func toTime(t int64) time.Time {
 	return storage.TimestampToTime(t)
 }
 
+func encodeTags(tagMap map[string]string) []*rpc.Tag {
```

Comment:
why not just have []rpc.Tag ?

Reply:
Required to conform to what protobuf expects